bevy_renet/src/transport.rs

```diff
@@ -22,18 +22,15 @@ impl Plugin for NetcodeServerPlugin {
     fn build(&self, app: &mut App) {
         app.add_event::<NetcodeTransportError>();
         app.configure_set(
+            Update,
             TransportSet::Server
                 .run_if(resource_exists::<NetcodeServerTransport>().and_then(resource_exists::<RenetServer>()))
                 .after(RenetSet::Server),
         );
 
-        app.add_system(Self::update_system.in_base_set(CoreSet::PreUpdate).in_set(TransportSet::Server));
-        app.add_system(Self::send_packets.in_base_set(CoreSet::PostUpdate).in_set(TransportSet::Server));
-        app.add_system(
-            Self::disconnect_on_exit
-                .in_base_set(CoreSet::PostUpdate)
-                .in_set(TransportSet::Server),
-        );
+        app.add_systems(PreUpdate, Self::update_system.in_set(TransportSet::Server));
+        app.add_systems(PostUpdate, Self::send_packets.in_set(TransportSet::Server));
+        app.add_systems(PostUpdate, Self::disconnect_on_exit.in_set(TransportSet::Server));
```

Comment:
Maybe group these systems in one call?
Similar for other places.

Reply:
Done!